### ISSUE TITLE
Mismatch of CUDA versions

### ISSUE THREAD

**Issue by KavyaRavulapati** [experiencor/keras-yolo2#378]
Hello, 
The code works fine on my   PU, but there's a mismatch of CUDA versions while running on GPU. The error is as follows:

 E tensorflow/stream_executor/cuda/cuda_dnn.cc:378] Loaded runtime CuDNN library: 7104 (compatibility version 7100) but source was compiled with 7004 (compatibility version 7000).  If using a binary install, upgrade your CuDNN library to match.  If building from sources, make sure the library loaded at runtime matches a compatible version specified during compile configuration.
2018-11-10 15:57:56.589793: F tensorflow/core/kernels/conv_ops.cc:717] Check failed: stream->parent()->GetConvolveAlgorithms( conv_parameters.ShouldIncludeWinogradNonfusedAlgo<T>(), &algorithms) 
Aborted (core dumped)

Current CUDA version on GPU: CUDA Version 9.2.148

Looks like the code runs with version 7. 

Don't know how to proceed! Kindly help!!

**Comment by rodrigo2019** [comment]
@KavyaRavulapati you should review your tensorflow instalation, this repo do not use any type of specific cuda version, it is pure python code that uses the modules installed in your machine.

**Comment by KavyaRavulapati** [comment]
@rodrigo2019 Yeah, but I'm trying to work on the GPU. Can you please elaborate

**Comment by rodrigo2019** [comment]
how did you installed the tensorflow lirabry?

**Comment by kdv4** [comment]
pip install tensorflow-gpu
